<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectdialog</class>
 <widget class="QDialog" name="selectdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>273</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>273</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>556</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>select</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>select</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rb_getart">
        <property name="text">
         <string>get aticles</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_remart">
        <property name="text">
         <string>remove aticels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_mids">
        <property name="placeholderText">
         <string>input categoties mid split by ','</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_cids">
        <property name="placeholderText">
         <string>input aticle cid split by ','</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>cids:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>mid:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
